Cue/app/tabs/library/play/PlayDeckSetupView.js

```diff
@@ -133,6 +210,7 @@ export default class PlayDeckSetupView extends React.Component {
         <ScrollView style={styles.container}>
```

Comment:
Can you add like, 40 pt of bottom padding to the `ScrollView`'s `contentContainerStyle` so that the bottom rows aren't exactly touching the bottom of the screen?

![screen shot 2017-03-25 at 6 17 36 pm](https://cloud.githubusercontent.com/assets/13400887/24326572/5e16968c-1187-11e7-9de2-5c0a2f2f765e.png)